### DISCUSSION TITLE
Add a light file to use across CDN

### DISCUSSION THREAD

**Discussion by ghost** [twbs/bootstrap#33160] [Q&A] [answered]
Hi I have a suggestion. Add a lightweight file to use across CDN
Like converting `mx-auto` From `margin-left:auto!important;margin-right:auto!important` to `margin:0 auto!important`
In order to increase the speed of the websites

**Comment by coliff** [comment, marked as answer]
`mx-auto` is only for zero margin on left and right.
If you want zero margin on top/bottom/left and right you can use `m-auto`.

So the rules for `mx-auto` are as minified as they can.
Hope that helps!
